kafka-connector-demo/pom.xml

```diff
@@ -6,6 +6,9 @@
   <artifactId>kafka-connector-demo</artifactId>
   <version>11.2.0-SNAPSHOT</version>
```

Comment:
I think we should indicate the version-bump to leading-edge 11.3 also with the versions within the connector artifact pom's. 
I'd raise them all to 11.3.0-SNAPSHOT